picard/ui/ui_options_script.py

```diff
@@ -108,7 +116,8 @@ def setupUi(self, ScriptingOptionsPage):
 
     def retranslateUi(self, ScriptingOptionsPage):
         _translate = QtCore.QCoreApplication.translate
-        self.enable_tagger_scripts.setTitle(_("Tagger Script(s)"))
+        self.enable_tagger_scripts.setTitle(_("Enable Tagger Script(s)"))
+        self.label.setText(_("<html><head/><body><p>Checked scripts will be executed automatically, once for each track in the release, when Picard retrieves release information from MusicBrainz.</p></body></html>"))
```

Comment:
Maybe "Activated" instead of "Checked"? But not sure. For me "Checked" just makes it look more like "Select", and that usually is done to do something with the selected items. But here it is clearly to activate / disable the script.

We could also call it "Enable", but that conflicts with the "Enable" of the entire feature, so it's confusing.

Reply:
How about something like, "Scripts that have been activated below will be executed automatically, once for each track in the release, when Picard retrieves release information from MusicBrainz."?

Reply:
Yes, good approach. It's technically not fully correct, as the script need also to be re-run also when matching files. But the goal is to have the scripts process the metadata of each loaded track, the rest is an implementation details. So maybe even simpler:

> Scripts that have been activated below will be executed automatically for each track of a release loaded from MusicBrainz.